example_scenes/advanced_tex_fonts.py

```diff
@@ -44,8 +43,9 @@ def construct(self):
 
 
 class TexFontTemplateLibrary(Scene):
-    """An example scene that uses TexTemplate objects from the TexFontTemplates collection
-    to create sample LaTeX output in every font that will compile on the local system.
+    """An example scene that uses TexTemplate objects from the TexFontTemplates
+    collection to create sample LaTeX output in every font that will compile on
+    the local system.
```

Comment:
```suggestion
    """An example scene that uses TexTemplate objects from the TexFontTemplates collection to create sample LaTeX output in every font that will compile on the local system.
```
I know this needs to be shorter, but I don't know how.